### DISCUSSION TITLE
a

### DISCUSSION THREAD

**Discussion by TheRealSunder** [jef/streetmerchant#1053] [Help] [answered]

**Comment by ryandh** [comment, marked as answer]
go to src/store/model

find the merchant, and insert new links there. 
you can just copy and paste, then replace the titles and links

![image](https://user-images.githubusercontent.com/5561911/100782081-c2362d80-33c0-11eb-86bb-90639693f59d.png)

then restart the app

**Reply by TheRealSunder** [reply]
Thanks.
